### PR TITLE
Update SE House form

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Update and translate most of SE House form.

Refactor select to use either label or placeholder.

## Justify why they are needed

I left extra building input update for a different PR.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1166** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1166" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1166?utm_source=stack-comment).

File: apps/store/src/services/PriceCalculator/data/SE_HOUSE.ts [hunk 1]
```diff
@@ -14,88 +14,88 @@ export const SE_HOUSE: Template = {
...
-              { label: { key: 'Attefall' }, value: 'ATTEFALL' },
-              { label: { key: 'Outhouse' }, value: 'OUTHOUSE' },
-              { label: { key: 'Guesthouse' }, value: 'GUESTHOUSE' },
-              { label: { key: 'Gazebo' }, value: 'GAZEBO' },
-              { label: { key: 'Greenhouse' }, value: 'GREENHOUSE' },
-              { label: { key: 'Sauna' }, value: 'SAUNA' },
-              { label: { key: 'Barn' }, value: 'BARN' },
-              { label: { key: 'Boathouse' }, value: 'BOATHOUSE' },
-              { label: { key: 'Other' }, value: 'OTHER' },
+              { label: { key: 'FIELD_EXTRA_BUIDLINGS_GARAGE_LABEL' }, value: 'GARAGE' },
```
Comment (alebedev) [inline apps/store/src/services/PriceCalculator/data/SE_HOUSE.ts:82]: Nit: can we use some sort of helper to generate values and labels without loosing type safety?
```
buildingOptions: [
  buildingOption('CARPORT'),
  buildingOption('SHED')
  // ...
]
```

File: apps/store/src/services/PriceCalculator/data/SE_HOUSE.ts [hunk 2]
```diff
@@ -14,88 +14,88 @@ export const SE_HOUSE: Template = {
...
+          layout: LAYOUT.FULL_WIDTH,
         },
         {
           field: {
             type: 'extra-buildings',
             name: 'extraBuildings',
-            label: { key: 'Extra buildings' },
+            label: { key: 'FIELD_EXTRA_BUILDINGS_LABEL' },
             defaultValue: [],
             buildingOptions: [
```
Comment (alebedev) [inline apps/store/src/services/PriceCalculator/data/SE_HOUSE.ts:81]: Explicitly stating options is each form does not look DRY.  Can we move them to field implementation or reusable fields module?

**Review by alebedev — APPROVED** [review]
Looks good, some questions inside

File: apps/store/src/components/InputSelect/InputSelect.tsx [hunk 3]
```diff
@@ -76,9 +79,9 @@ export const InputSelect = ({
             {...rest}
             {...animationProps}
           >
-            {placeholder && (
+            {labelText && (
               <Placeholder value="" disabled selected>
```
[the remark this answers is not in the document]
Comment (robinandeer) [reply apps/store/src/components/InputSelect/InputSelect.tsx:83]: For select we use a disabled option as both placeholder and label

But I agree that we can clean this up

File: apps/store/src/services/PriceCalculator/data/SE_HOUSE.ts (hunk 2 above)
Comment (robinandeer) [reply apps/store/src/services/PriceCalculator/data/SE_HOUSE.ts:81]: Yep sounds good - at least if they are shared across countries

File: apps/store/src/services/PriceCalculator/data/SE_HOUSE.ts (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/services/PriceCalculator/data/SE_HOUSE.ts:82]: I think adding this to a fields module would hide enough complexity, don't you think?

Comment (alebedev) [reply apps/store/src/services/PriceCalculator/data/SE_HOUSE.ts:82]: Agree, we can always iterate on it later if we ever get country-specific list of building types

**Pull request merged**